scootapi/testHelpers/generators.go

```diff
@@ -0,0 +1,51 @@
+package testHelpers
```

Comment:
can you rename the package to be all lowercase?

https://blog.golang.org/package-names
https://golang.org/doc/effective_go.html#names